Sources/Starknet/Data/Felt.swift

```diff
@@ -41,6 +41,10 @@ public struct Felt {
     public func toHex() -> String {
         "0x\(String(value, radix: 16))"
     }
+
+    public var debugDescription: String {
+        "Felt: \(self.toHex())"
+    }
```

Comment:
Should we keep that or was it added for testing purposes only?

Reply:
For testing purposes, but I think we can keep it unless someone has specific reasons not to